### ISSUE TITLE
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-PMnen1/simpleaudio/

### ISSUE THREAD

**Issue by Sispheor** [hamiltron/py-simple-audio#12]
I got this on Ubuntu 16.04. When I run the pip installation. 

```
Collecting simpleaudio
  Downloading simpleaudio-1.0.1.tar.gz (2.0MB)
    100% |████████████████████████████████| 2.0MB 794kB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-PMnen1/simpleaudio/setup.py", line 40, in <module>
        encoding='utf-8') as f:
    TypeError: 'encoding' is an invalid keyword argument for this function
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-PMnen1/simpleaudio/
```

**Comment by hamiltron** [comment]
Based on the reported command, it looks like this may be attempting to run using the Python 2 executable (which will not work).

**Comment by Sispheor** [comment]
Oh! Totally right. We are in the Python 3 area now ;). No issue actually.
Sorry for the disturbing and thank you for your answer. Closes #12

**Comment by hamiltron** [comment]
No worries! Glad you got it sorted. Happy coding. :)
